Source/WebSocket/WinRT/winrt_websocket.cpp

```diff
@@ -282,7 +282,7 @@ HRESULT CALLBACK Internal_HCWebSocketConnectAsync(
 {
     UNREFERENCED_PARAMETER(env);
     std::shared_ptr<winrt_websocket_impl> websocketTask = std::make_shared<winrt_websocket_impl>();
-    websocketTask->m_websocketHandle = HCWebSocketDuplicateHandle(websocket);
```

Comment:
>HCWebSocketDuplicateHandle [](start = 39, length = 26)

I think it is also called in send.

Reply:
Got it

Reply:
Where did this get closed anyhow? do we need to remove that?

---
In reply to: [284028839](https://github.com/microsoft/libHttpClient/pull/415#discussion_r284028839) [](ancestors = 284028839)

Reply:
Right, good call.  The closes need to be removed as well